mos_tests/object_storage/test_crud_operations.py

```diff
@@ -0,0 +1,298 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+import os
+import random
+import re
+import yaml
+
+import pytest
+from six.moves import configparser
+from tempest.lib.cli import output_parser as parser
+
+
+logger = logging.getLogger(__name__)
+
+
+def templates_dir_path():
+    """Returns full path to template dir"""
+    return os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), 'templates/')
+
+
+def wa_for_1543135(env):
```

Comment:
Let's use `openstack` CLI client (it doesn't need WA, also keystone client recommends to use it)